public/javascripts/tests.js

```diff
@@ -1,6 +1,24 @@
 var is_operator;
 var restart_url;
 
+function highlight_jobs ( enable, children, parents ) {
+    if (enable) {
+        for (i = 0; i < children.length; ++i) $("#job_" + children[i]).addClass('highlight_child');
+        for (i = 0; i < parents.length; ++i) $("#job_" + parents[i]).addClass('highlight_parent');
+    }
+    else {
+        for (i = 0; i < children.length; ++i) $("#job_" + children[i]).removeClass('highlight_child');
+        for (i = 0; i < parents.length; ++i) $("#job_" + parents[i]).removeClass('highlight_parent');
+    }
+}
+
+
+function highlight_jobs_html (children, parents) {
+    return ' onmouseover="highlight_jobs(true, [' + children.toString() + '], [' + parents.toString() + '])" ' +
+           ' onmouseout="highlight_jobs(false, [' + children.toString() + '], [' + parents.toString() + '])" ';
+}
+
```

Comment:
please solve this using jquery instead of onmouse* attributes: http://api.jquery.com/mouseover/

You put the children as data (without toString()) into the node and then bind it the functions